### ISSUE TITLE
Slow OBJ import after OBJ line break commit

### ISSUE THREAD

**Issue by dhritzkiv** [assimp/assimp#1237]
Ever since commit 9fc7f72b551ee160eef96583a968b9cf981df630, I've noticed import times increase 4 orders of magnitude when importing certain OBJs.

The command each time:

```
> assimp export guitar.obj guitar.gltf --triangulate --find-degenerates
```

Results before commit:

```xterm
assimp export: select file format: 'gltf' (GL Transmission Format)
Launching asset import ...           OK
Validating postprocessing flags ...  OK
Importing file ...                   OK 
   import took approx. 0.09498 seconds

Launching asset export ...           OK
Exporting file ...                   OK 
   export took approx. 0.10536 seconds

assimp export: wrote output file: guitar.gltf
```

Results using latest (4c348a6cdc5c35972af640aee967044f42608dab) at build time:

```xterm
assimp export: select file format: 'gltf' (GL Transmission Format)
Launching asset import ...           OK
Validating postprocessing flags ...  OK
Importing file ...                   OK 
   import took approx. 2378.59967 seconds

Launching asset export ...           OK
Exporting file ...                   OK 
   export took approx. 0.31183 seconds

assimp export: wrote output file: guitar.gltf
```

Makefile generated with `cmake . -DCMAKE_BUILD_TYPE=release`

OS: macOS 10.12.4
Specs: 3.4GHz Intel Core i7

Here is the example file: [guitar.obj.zip (5Mb)](https://github.com/assimp/assimp/files/944002/guitar.obj.zip)

**Comment by kimkulling** [comment]
This is realy bad, thanks for figuring this out!

**Comment by samitc** [comment]
I noticed the performance issue and fix it.
sorry for that.
Here is the PR #1254

**Comment by dhritzkiv** [comment]
This issue is fixed in #1254 👍

**Comment by kimkulling** [comment]
Great, thx for all your help!

Daniel Hritzkiv <notifications@github.com> schrieb am Do., 4. Mai 2017,
20:25:
